src/operators.jl

```diff
@@ -111,15 +111,22 @@ julia> DifferentialOperator(x̄, (Reflecting(), Reflecting()), CentralSecondDiff
 """
 function DifferentialOperator(x̄, bc::Tuple{Reflecting, Reflecting}, method::DifferenceMethod)
     T = eltype(x̄)
+    Δ_1m = x̄[2] - x̄[1]
+    Δ_1p = x̄[3] - x̄[2]
+    Δ_Mm = x̄[end-1] - x̄[end-2]
+    Δ_Mp = x̄[end] - x̄[end-1]
 
     # get basis operator on interior nodes
     L = get_basis_operator(x̄, method)
-
+    
+    Ξ_1 = -2*(1/(Δ_1m*Δ_1p)-1/((Δ_1p+Δ_1m)*(Δ_1m)))
```

Comment:
Are these correct?

Reply:
Yup, I'll create a separate issue to work on full derivation of $L_2^B$ and $L_{1+}^B$.